planner/property/physical_property.go

```diff
@@ -51,8 +53,25 @@ const (
 	BroadcastType
 	// HashType requires current task to shuffle its data according to some columns.
 	HashType
+	// SinglePartitionType requires all the task pass the data to one node (tidb/tiflash).
+	SinglePartitionType
```

Comment:
to merge? more clear and compitable with the future work

Reply:
I think this name is enough clear and compitable, for example, we can add more type like "multiPartition", etc.

at least, this won't effect user interface and can be changed easily.